dust_attenuation/averages.py

```diff
@@ -118,7 +118,7 @@ def k_lambda(self, x):
 
         axEbv[nir_indxs] = 2.659 * (-1.857 + 1.040 * 1 / x[nir_indxs]) + self.Rv
 
-        return axEbv
+        return np.maximum(axEbv, 0.)
```

Comment:
This is not something I had thought about - negative attenuations.  Did you get such and this alerted you to this issue?  I wonder if a warning or error would be more appropriate than silently setting the results to 0?